benchmarks/tests/test_tpch_benchmark.py

```diff
@@ -0,0 +1,96 @@
+import copy
+
+from .. import tpch_benchmark
+from ..tests import _asserts
+
+HELP = """
+Usage: conbench tpch [OPTIONS]
+
+  Run tpch benchmark(s).
+
+  For each benchmark option, the first option value is the default.
+
+  Valid benchmark combinations:
+  --query-num=1 --scale=1 --format=native
```

Comment:
@jonkeane @dianaclarke Do we want to have consistent TPCH option names with voltrondata/benchmarks repo that uses `query-id` and `sf` instead of `query-num` and `scale`?

Reply:
I wondered that too. I can change these without the R needing to change. Thanks!

Reply:
Yeah, I agree I'm happy to change the R (and should be able to do that quickly) to line up with everything, but agree with both: `query_id` for the id number of the query and `scale_factor` for the scale factor across both. I've used `_` instead of `-` as a divider since at least for R `-` in option names doesn't work (or is a big hassle with special quoting needed). I don't mind if we use `_` some places and `-` others, though _mostly_ I see `_` in other tag names

Reply:
- `-` (dash) is only used at the CLI
- `_` (underscore) is always used in the python
- `_` (underscore) should be used in the Conbench tags

Reply:
PS. I also tag `memory_map` rather than `mem_map`.

Reply:
Thanks for the dash-underscore clarity there that's what I was flailing at without having the words, clarity, or structure that you did. I'm all on board.

re: `memory_map` that's good, I'll do the same